src/main/kotlin/com/atlassian/performance/tools/report/jstat/JstatConverter.kt

```diff
@@ -15,7 +15,10 @@ internal class JstatConverter {
     ): Path {
         val jstatCsvName = "${jstat.fileName.toString().substringBeforeLast(".")}.csv"
         val jstatCsv = jstat.parent.resolve(jstatCsvName)
-        jstatCsv.toFile().createNewFile()
+        jstatCsv.toFile().let {
+            it.delete()
+            it.createNewFile()
```

Comment:
Is it needed at all? If we start writing to the path, the file will appear.

Reply:
Dunno. It was here before.

Reply:
I deleted this and it still works.